### PR TITLE
Admin invoices cont

### PR DESCRIPTION
**Pull request by ajpulzone**
*I pulled down main prior to pull request 25 and resolved any merge conflicts

Pull request:
1.) Added drop down menu for invoice status selection, added functionality to updated that status with a button on the page.
2.) Added Admin Dashboard partial and applied that partial to all admin views
3.) Refactored all admin views to match the pictures we were given as guides.

### CODE REVIEW

File: app/controllers/admin/invoices_controller.rb [hunk 1]
```diff
@@ -7,5 +7,16 @@ def index
   def show
     @invoice = Invoice.find(params[:id])
   end
+
+  def update
+    @invoice = Invoice.find(params[:id])
+    @invoice.update(invoice_params)
+    @invoice.save
```
Comment (Freeing3092) [inline app/controllers/admin/invoices_controller.rb]: This method works great, but I don't believe the `save` method is necessary here as `update` should save for us.

File: app/controllers/admin/invoices_controller.rb [hunk 2]
```diff
@@ -7,5 +7,16 @@ def index
...
+    redirect_to admin_invoice_path(@invoice.id)
+  end
+
+  def invoice_params
```
Comment (Freeing3092) [inline app/controllers/admin/invoices_controller.rb:18]: I believe we would want to specify this as a `private` method for security.

**Review by Freeing3092 — CHANGES_REQUESTED** [review]
This  looks great. I like the addition of the partial to DRY up the views.

The only comment that I think should be addressed before merging to main relates to making the strong params a private method for security purposes.

**Comment by ajpulzone** [comment]
Pushed up branch with the requested changes made. AJP

**Pull request merged**